platform/client-admin/src/pages/ViewRecipientPage.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { RouteComponentProps } from 'react-router-dom'
+import { Header } from '../components/Header';
+import { IUpdateMatchParams } from '../declarations';
+import { useFindRecipientsQuery, useUpdateRecipientMutation } from '../dataFacade'
+import { AutoForm } from 'uniforms-ionic'
+import recipientFormSchema from '../forms/recipient';
+import { IonLoading, IonContent, IonList, IonCard, IonItemGroup, IonItemDivider } from '@ionic/react';
+
+export const ViewRecipientPage: React.FC<RouteComponentProps<IUpdateMatchParams>> = ({ match }) => {
+  const { data, loading, error } = useFindRecipientsQuery({ fetchPolicy: "network-only", variables: { fields: {id: match.params.id}, limit: 1 } });
+const [updateRecipient]= useUpdateRecipientMutation()
+  if (error) {
+    console.log(error);
+  }
+
+  if (!data || !data.findRecipients[0]) {
+    return <div>Cannot fetch element with provided id</div>
+  }
+
+  if (loading) return <IonLoading isOpen={loading} message={'Loading...'} />;
+  const model = data.findRecipients[0];
+
+  return (
+    <>
+      <Header title="Manage Recipient" match={match} />
+      <IonContent>
+        <IonList>
+          <IonCard>
+            <IonItemGroup>
+              <IonItemDivider color="light">
+                <h2>Recipient information</h2>
+              </IonItemDivider>
+              <AutoForm
+                placeholder
+                model={model}
+                schema={recipientFormSchema}
+                onSubmit={(model: any) => {
+                  updateRecipient({
+                    variables: {
+                      input: {
+                        id: model.id,
```

Comment:
Model for recipient is quite bad. I think we can add some improvements. 
Usually we would do Address object and use it in all models but that is limitation of graphback (we cannot have object queried without relationship so we can just keep the same field by using interface etc.

I would also introduce products model with some basic information. 
This will allow us to introduce stock numbers in the distribution centre and link them to actions